gramhopper/users_helper.py

```diff
@@ -17,8 +17,9 @@ def __init__(self, file_path: Union[PathLike, str, bytes] = None):
 
     def load_users(self, force: bool = False):
         if force or not self._users:
-            with open(self.file_path, 'r') as file:
-                self._users = json.load(file)
+            if os.path.exists(self.file_path):
+                with open(self.file_path, 'r') as file:
+                    self._users = json.load(file)
```

Comment:
What happens if such file doesn't exist? No users will be defined?

Reply:
After this change? Yes. No users will be defined (the dictionary of users will be empty).

Reply:
K